src/ocm/components/clusterWizard/ClusterWizardContextProvider.tsx

```diff
@@ -63,10 +127,12 @@ const ClusterWizardContextProvider: React.FC<
       if (!staticIpInfo) {
         throw `Wizard step is currently ${currentStepId}, but no static ip info is defined`;
       }
-      const newStepIds = getWizardStepIds(staticIpInfo.view);
-      if (!isEqual(newStepIds, wizardStepIds)) {
-        setWizardStepIds(newStepIds);
-      }
+      const newStepIds = getWizardStepIds(
+        wizardStepIds,
+        staticIpInfo.view,
+        addCustomManifests || (customManifests && customManifests.length > 0),
```

Comment:
This action is about static IP, so it shouldn't need passing anything about custom manifests